packages/pando-repository/app/src/script.js

```diff
@@ -156,3 +141,25 @@ const fetchFilesFromTree = (hash, path) => {
     })
   })
 }
+
+const fetchOrderedHistory = async (hash, formerHistory) => {
```

Comment:
In the final version of this PR, we'll deprecate all of these helper functions in this file and import them from `pando.ts`